content/post/52-go-ipfs-0.4.18.md

```diff
@@ -0,0 +1,283 @@
+---
+date: 2018-11-01
+url: 52-go-ipfs-0.4.18
+title: go-ipfs 0.4.18 released
+author: Steven Allen and Erik Ingenito
+---
+
+[go-ipfs 0.4.18](https://dist.ipfs.io/#go-ipfs) has been released! This is one of largest go-ipfs releases to date; 3 months in the making. *Thanks to all our contributors for your awesome work!*
+
+## Highlights
+The headline features this release are experimental [QUIC support](#quic), the [gossipsub](#pubsub)
+pubsub routing algorithm, [pubsub message signing](#pubsub), a refactored `ipfs p2p`
+[command](#commands-changes), and a new [WebUI](#webui). However, that's just scratching the surface.
+
+
+## QUIC
+First up, on the networking front, this release has also introduced experimental
```

Comment:
"has also" is very passive - just drop! We're also using present tense throughout =]
> "...on the networking front, this release **introduces** experimental support for..."